decentralized_devstack/provision-lms.sh

```
@@ -26,7 +26,11 @@ log_step "lms: Creating MongoDB users..."
 docker-compose exec -T mongo bash -c "mongo" < decentralized_devstack/provision-mongo.js
 
 log_step "lms: Adding default MongoDB data..."
-service_exec mongo mongorestore --gzip /data/dump
+# FYI, without --quiet flag below, when this command is run on database which was previously provisioned,
+# it will throw out errors complaining about duplicates(E11000 duplicate key error collection),
+# we are  ignoring these errors(using --quiet flag),
```

Nit: I don't think we're really ignoring the errors -- the `--quiet` flag should only suppress info messages, not error messages or error exit code.

---

@timmc-edx I removed alot from this, what do you think of the rewording?